src/Raven.Client/Documents/Operations/Revisions/RevertDocumentsToRevisionsOperation.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using Raven.Client.Documents.Commands;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Sparrow.Json;
+
+namespace Raven.Client.Documents.Operations.Revisions
+{
+    public class RevertDocumentsToRevisionsOperation : IOperation
```

Comment:
can we come up with a better name? My thinking is 'to what elese we can revert a document'? Maybe just `RevertDocumentsOperation` ? cc @karmeli87 Thoughts?

Reply:
We already have `RevertRevisionsOperation` which reverts collections, so to be somewhat consistent maybe `RevertRevisionsByIdOperation` ?

Reply:
Done